src/routes/live_ov.route.js

```diff
@@ -0,0 +1,90 @@
+const OpenVidu = require('openvidu-node-client').OpenVidu;
+
+// Environment variable: URL where our OpenVidu server is listening
+const OPENVIDU_URL = 'localhost:4443';
```

Comment:
This won't work in production, you should replace it with `https://162.38.114.61:4443`.